src/js/tabs/firmware_flasher.js

```diff
@@ -983,7 +983,7 @@ TABS.firmware_flasher.initialize = function (callback) {
             if ($('option:selected', this).data().isDFU) {
                exitDfuElement.removeClass('disabled');
             } else {
-                exitDfuElement.addClass('disabled');
+               exitDfuElement.addClass('disabled');
```

Comment:
This brings the line out of alignment actually. :-)

Reply:
Ouch, then does this white space revert needed?

Reply:
Ok fixed now